Client/Shared/TopBarComponent.razor

```diff
@@ -25,31 +27,36 @@
                 Ask question
             </div>
         </NavLink>
-        <div class="icon icon-mail @(isWhiteTheme ? "" : "icon-dark")"/>
-        <div class="icon icon-notification @(isWhiteTheme ? "" : "icon-dark")"/>
+        <div class="icon icon-mail"/>
+        <div class="icon icon-notification"/>
         <img src="@Avatar" class="profile-logo"/>
-
     </Authorized>
 </AuthorizeView>
 
 
 @code {
 
-    /// <summary>
-    ///     Zmienna informująca jaki jest aktualny theme
-    /// </summary>
-    [CascadingParameter]
-    protected bool isWhiteTheme { get; set; }
 
     /// <summary>
     ///     Zmienna przechowująca Awatar
     /// </summary>
     private string Avatar;
 
-
     protected override async Task OnInitializedAsync()
     {
         Avatar = await _localStorageService.GetItemAsync<string>("avatar");
+        _navManager.LocationChanged += MyListener;
+        Console.WriteLine("IA:" + Avatar);
+    }
+
+    private void MyListener(object? sender, LocationChangedEventArgs e)
+    {
+        Console.WriteLine("called: " + sender + " ; " + e.IsNavigationIntercepted);
```

Comment:
Is it necesarry to write to console?

Reply:
No it's not. I just forgot to remove it 😄